config/jobs/kubernetes-sigs/cluster-api/cluster-api-periodics-release-1-4-upgrades.yaml

```diff
@@ -46,17 +45,20 @@ periodics:
         resources:
           requests:
             cpu: 7300m
+            memory: 32Gi
+          limits:
+            cpu: 7300m
+            memory: 32Gi
```

Comment:
Do we need so much memory ? cc @killianmuldoon

Reply:
I think we picked this because we got the info that the Nodes in that cluster have a pretty high memory to CPU ratio (16 CPU 128 GB memory => 1:8) and we wanted to tune it later.

For now the idea was to just avoid any risk of OOM kills to not delay the move to EKS